civictechprojects/templates/new_index.html

```diff
@@ -19,6 +19,24 @@
     <link rel="stylesheet" type="text/css" href="/static/css/SubHeader.css"/>
   </head>
   <body>
+    <script>
+      window.fbAsyncInit = function() {
+        FB.init({
+          appId      : '137258853647008',
```

Comment:
Should this appId be factored out into an environment variable, so traffic from people's development environments don't pollute the results?

Reply:
Totes see your concern -- we should be okay because we can create a segment (filter) when looking at data to filter our localhost, e.g.: 

![image](https://user-images.githubusercontent.com/3479724/33800955-dfa9869c-dd00-11e7-8e28-273d152f2aef.png)

Reply:
Okay, that should be good enough for the present.